sqlmesh/core/engine_adapter/athena.py

```diff
@@ -66,10 +66,8 @@ def create_state_table(
             table_name,
             columns_to_types,
             primary_key=primary_key,
-            table_properties={
-                # it's painfully slow, but it works
-                "table_type": exp.Literal.string("iceberg")
-            },
+            # it's painfully slow, but it works
+            table_format="iceberg",
```

Comment:
Promoting it to a top-level property makes it explicit and consistent across adapters